svc/pkg/domain/model/lgtm.go

```diff
@@ -1,7 +1,8 @@
 package model
 
+import "time"
+
 type LGTM struct {
-	ID string
+	ID        string
```

Comment:
一応 firestore tagをつけておいたほうが良さそう..!
Firestore上ではこのIDは無視するという認識であっている??
その場合だと、以下の形で行けそう!
```suggestion
	ID        string `firestore:"-"`
```